Library/Formula/gtk+.rb

```diff
@@ -27,6 +27,12 @@ class Gtkx < Formula
     cause "Undefined symbols when linking"
   end
 
+  def patches
+    # Patch from Bug 557780 - Missing support for depth == 1 in gdk_image_new_for_depth()
+    # https://bugzilla.gnome.org/show_bug.cgi?id=557780
+    DATA
+  end
```

Comment:
Neat trick!  I didn't know about that.

I have uploaded a new patch with a slightly different URL to avoid a missing diff header issue.